types/networks/src/lib.rs

```diff
@@ -131,6 +64,10 @@ pub struct HeightInfo {
     pub epoch: ChainEpoch,
 }
 
+pub fn normalize(height_info_vec: &mut [HeightInfo]) {
```

Comment:
`sort_by_epoch` would be a more meaningful name here. Also, it would be cleaner to get rid of those references and just take by value and return by value since you're cloning the `height_infos` anyway.

Reply:
Fixed